pkg/cloud/services/compute/instance.go

```diff
@@ -579,7 +579,7 @@ func (s *Service) DeleteInstance(eventObject runtime.Object, instanceStatus *Ins
 				return nil
 			}
 
-			s.scope.Logger().Info("deleting dangling root volume %s(%s)", volume.Name, volume.ID)
+			s.scope.Logger().Info("Deleting dangling root volume", "volumeName", volume.Name, "volumeID", volume.ID)
```

Comment:
I'm wondering if the logging should be moved out of this function to the calling function. And then it should log a line after calling `getOrCreateRootVolume`, using past tense.
E.g. "deleted volume". 

This is something for a separate PR, however requires us to rethink where we log and where not.